src/daemons/rhsmcertd.c

```diff
@@ -164,9 +185,10 @@ r_log (const char *level, const char *message, ...)
         log_file = stdout;
         use_stdout = true;
     }
+    format_timestamp (timestamp, sizeof timestamp);
```

Comment:
To improve readability
```suggestion
    format_timestamp (timestamp, sizeof (timestamp));
```